news_items/004_freq_testing_paper/index.html

```diff
@@ -0,0 +1,54 @@
+<div class="d-flex justify-content-center">
+  <div class="card mb-4" style="width: 80%;">
+    <div class="card-body bg-light">
+      Morphew, J. W., Silva, M., Herman, G., and West, M., <b>"Frequent mastery testing with second-chance exams leads to enhanced student learning in undergraduate engineering"</b>, <i>Applied Cognitive Psychology</i> 34(1), 2020. DOI: <a href="https://doi.org/10.1002/acp.3605">10.1002/acp.3605</a>. Full-text link: <a href="http://lagrange.mechse.illinois.edu/pubs/MoSiHeWe2020/MoSiHeWe2020.pdf">PDF</a>.
+    </div>
+  </div>
+</div>
+
+<p>
+  In this recently published paper, <a href="https://engineering.purdue.edu/ENE/People/ptProfile?resource_id=222158">Jason Morphew</a> and co-authors showed that frequent short PrairieLearn exams in the CBTF (Computer-Based Testing Facility) substantially increased student learning in TAM 251 ("Introductory Solid Mechanics") at the University of Illinois. Most impressively, these learning gains were on an identical pen-and-paper final exam and there were improvements at all grade levels.
+</p>
+
+<p>
+  <b>How did TAM 251 use the CBTF?</b> In previous semesters the course used a <i>traditional exam schedule</i> with two pen-and-paper midterm exams. <a href="https://mfsilva.web.illinois.edu">Mariana Silva</a> converted it to a <i>frequent exam schedule</i> with seven short CBTF exams and seven optional second-chance retry exams (see Figure 1). The new exams were all implemented in PrairieLearn and allowed students to re-attempt questions for partial credit, with a mastery grading scheme that only awarded points for correct answers.
```

Comment:
> "re-attempt questions for partial credit"

What was the partial credit scheme? How many re-attempts inside each question unit?

Reply:
Any chance to "show" a photo of a question one would experience during a TAM course? (Practice question to avoid burning one in the test bank?)

Reply:
I'm deliberately not going into the details of the partial credit scheme, as it varied by question type and is way too hard to summarize without being inaccurate.

Reply:
We included example questions in the paper, which I hope people will read if they are interested in the details! 😄